console-subscriber/examples/local.rs

```diff
@@ -0,0 +1,28 @@
+use std::time::Duration;
```

Comment:
can we include a quick doc comment or something in this example explaining what it demonstrates? not a big deal.

Reply:
Good point. Done.